packit/config/package_config.py

```diff
@@ -194,7 +194,26 @@ def load_packit_yaml(
         return safe_load(raw_text) or {}
     except YAMLError as ex:
         logger.error(f"Cannot load package config {config_file_path}.")
-        raise PackitConfigException(f"Cannot load package config: {ex!r}.")
+        if hasattr(ex, "problem_mark"):
+            if ex.context is not None:  # type: ignore
+                logger.error(
+                    "  parser says\n"
+                    + str(ex.problem_mark)
+                    + "\n  "
+                    + str(ex.problem)  # type: ignore
+                    + " "
+                    + str(ex.context)  # type: ignore
+                )
+            else:
+                logger.error(
+                    "  parser says\n"
+                    + str(ex.problem_mark)
+                    + "\n  "
+                    + str(ex.problem)  # type: ignore
+                )
+        else:
+            logger.error(f"Cannot load package config {config_file_path}.")
```

Comment:
There already is one such logger.error above so there would be two if the `if hasattr(ex, "problem_mark")` is false.
```suggestion
```

Reply:
Right. I will put there the original error.